src/util.js

```diff
@@ -363,8 +363,9 @@ var ret = {
     markAsOriginatingFromRejection: markAsOriginatingFromRejection,
     classString: classString,
     copyDescriptors: copyDescriptors,
-    hasDevTools: typeof chrome !== "undefined" && chrome &&
-                 typeof chrome.loadTimes === "function",
+    hasDevTools: false,
+    // hasDevTools: typeof chrome !== "undefined" && chrome &&
+    //              typeof chrome.loadTimes === "function",
```

Comment:
You can link to `HAD-1673 Add artificial delay to promises to weed out timing issues causing test instability`